### DISCUSSION TITLE
[결과 분석] 긴 이미지

### DISCUSSION THREAD

**Discussion by jo-member** [bcaitech1/p4-ocr-hansarang#21] [General]
1. 긴 이미지를 못맞춘다
- 우리 default yaml 파일이 resize를 128,128로 시켜주기 때문에
```
transformed = transforms.Compose(
        [
            # Resize so all images have the same size
            transforms.Resize((options.input_size.height, options.input_size.width)),
            transforms.ToTensor(),
        ]
    )
```
위 augemtentation에 resize를 하나의 scale로 해주지 말고 우리의 data 분포에 따라 multiscale로 list의 형태로 넣어주면 해결될듯

**Comment by remaindere** [comment]
인상적이네요
![image](https://user-images.githubusercontent.com/48322490/120209017-21d69e00-c269-11eb-8582-ca79a4f3d752.png)
이랬던 이미지가

![image](https://user-images.githubusercontent.com/48322490/120208999-1b482680-c269-11eb-900b-2a26adaa72a6.png)
이래버리니

사람도 못 읽겠네요 

모델의 예측 결과인데
= \frac { \left( n + 2 \right) \left( a + 1 \right) } { 2 } = 7 8 = 2 \times 3 = 2 \times 3 3 3 \times 
실제로 사람이 알아볼 수 있을법한 2x39 까지 비슷하게 맞추고
그 뒤는 사람도 못맞추는 영역으로 가버리니 예측이 이상해지네요
